### DISCUSSION TITLE
Test of the Tedee Go connected lock + accessories - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#55] [Comments]
# Test of the Tedee Go connected lock + accessories - Haade.fr

Complete test and integration into the home-assistant of the Tedee Go lock with the Tedee wifi bridge and the Tedee access code keyboard

https://haade.fr/en/blog/test-of-the-tedee-go-connected-lock-and-accessories

**Comment by oywino** [comment]
Great writeup! 👍
Do you now why the Forum is dead empty?  https://tedee.freshdesk.com/support/home

**Reply by haade-administrator** [reply]
Sorry, i don't know
